backend/de.metas.handlingunits.base/src/main/sql/postgresql/ddl/functions/hu_label.sql

```diff
@@ -0,0 +1,68 @@
+DROP FUNCTION IF EXISTS "de.metas.handlingunits".hu_label(
+    p_M_HU_ID numeric
+)
+;
+
+
+CREATE OR REPLACE FUNCTION "de.metas.handlingunits".hu_label(
+    p_M_HU_ID numeric
+)
+    RETURNS TABLE
+            (
+                prod_name         character varying,
+                prod_value        character varying,
+                prod_desc         text,
+                displayableqrcode text,
+                renderedqrcode    text,
+                vendorName        text,
+                PurchaseOrderNo   text,
+                receiptdate       timestamp WITHOUT TIME ZONE,
+                proddate          date
+            )
+    STABLE
+    LANGUAGE sql
+AS
+$$
+SELECT DISTINCT p.name,
+                p.value            AS prod_value,
+                p.description      AS prod_desc,
+                qr.displayableqrcode,
+                qr.renderedqrcode,
+                ri.vendorName,
+                ri.PurchaseOrderNo,
+                ri.receiptdate,
+                tu.created :: date AS proddate
+FROM M_HU tu
+
+         /** Get Product */
+         INNER JOIN M_HU_Storage tus ON tu.M_HU_ID = tus.M_HU_ID
+         INNER JOIN M_Product p ON tus.M_Product_ID = p.M_Product_ID
+    -- QR Code
+         INNER JOIN M_HU_QRCode qr ON tu.m_hu_id = qr.m_hu_id
+
+    /** receipt infos */
+         LEFT OUTER JOIN (
+    SELECT DISTINCT ON (tu.M_HU_ID) tu.M_HU_ID,
+                                    bp.name         AS vendorName,
+                                    o.DocumentNo    AS PurchaseOrderNo,
+                                    io.movementdate AS receiptdate
+    FROM M_HU tu
+             LEFT OUTER JOIN M_HU_Item lui ON lui.M_HU_Item_ID = tu.M_HU_Item_Parent_ID AND lui.isActive = 'Y'
+             LEFT OUTER JOIN M_HU lu ON lu.M_HU_ID = lui.M_HU_ID
+             LEFT OUTER JOIN M_HU_PI_Version piv
```

Comment:
if we print the label to a random HU - why would we *not*  want to print the infos from the  the HU's `M_HU_PI_Version` if that piv is inactive? it's not like the HU has a better, more recent one, right?